userguide/text_classifier/introduction.md

```diff
@@ -71,3 +71,37 @@ classifier model using:
 ```python
 classifier = model.classifier
 ```
+
+##### Deploying to Core ML
+
+You can use the trained sentence classifier in Xcode by exporting it to Core ML. Exporting is done by:
+
+```python
+model.export_coreml('MySentenceClassifier.mlmodel')
+```
+
+Dragging the saved model into Xcode and inspecting it looks like the following:
+<img src="images/sentence_calssifier_model.png"></img>
+
+To use the model in Swift we can use the code below. Note that the model expects a bag-of-words representation of the input text.
+
+```swift
+let bagOfWords = bow(text: text)
```

Comment:
I think we should use [NSLinguisticTagger](https://developer.apple.com/documentation/foundation/nslinguistictagger/tokenizing_natural_language_text) in the example for tokenizing strings into bag-of-words.